sky/data/storage.py

```diff
@@ -1250,7 +1250,8 @@ def _delete_gcs_bucket(self, bucket_name: str) -> None:
             with backend_utils.safe_console_status(
                     f'[bold cyan]Deleting [green]bucket {bucket_name}'):
                 if num_files >= _GCS_RM_MAX_OBJS:
-                    remove_obj_command = f'gsutil -m rm -a gs://{bucket_name}/*'
+                    remove_obj_command = ('gsutil -m rm -a'
+                                          f' gs://{bucket_name}/**')
```

Comment:
Does this delete the bucket after it's become empty, or leave an empty bucket there?

Also, write a tested section? Try two cases of the bucket: (1) `bucket/{file1, file2}` (2) with nested levels, `bucket/file, bucket/dir/dir2/file, bucket/dir2/file`, etc.

Reply:
Great question, it will delete the objects/folders and then delete the empty bucket:
https://github.com/skypilot-org/skypilot/blob/deb089dabf46485793f35992ed33999dc64cd6b8/sky/data/storage.py#L1250-L1255

Aight, will make a test.

Reply:
Done, added test.